src/renderer/components/ConfigureBeaconNode/ConfigureBeaconNode.tsx

```diff
@@ -1,30 +1,112 @@
-import React, {useState} from "react";
+import React, {useRef, useState} from "react";
+import path from "path";
 import {DockerPort} from "../../services/docker/type";
-import {getNetworkConfig} from "../../services/eth2/networks";
+import {getNetworkConfig, networksList} from "../../services/eth2/networks";
 import {ButtonPrimary} from "../Button/ButtonStandard";
 import {InputForm} from "../Input/InputForm";
+import {Dropdown} from "../Dropdown/Dropdown";
+import {remote} from "electron";
+import {getConfig} from "../../../config/config";
 
-interface IConfigureBNProps {
+export interface IConfigureBNPSubmitOptions {
+    ports: DockerPort[];
     network: string;
-    onSubmit: (ports: DockerPort[], libp2pPort: string, rpcPort: string) => void;
+    folderPath: string;
+    eth1Url: string;
+    discoveryPort: string;
+    libp2pPort: string;
+    rpcPort: string;
+}
+
+interface IConfigureBNProps {
+    onSubmit: (values: IConfigureBNPSubmitOptions) => void;
 }
 
 export const ConfigureBeaconNode: React.FunctionComponent<IConfigureBNProps> = (props: IConfigureBNProps) => {
-    const ports = getNetworkConfig(props.network).dockerConfig.ports;
-    const defaultRpcPort = ports[1].local;
+    // TODO: refactor to use list from src/renderer/services/eth2/networks/index.ts
+    const [networkIndex, setNetworkIndex] = useState(0);
+    const ports = getNetworkConfig(networksList[networkIndex]).dockerConfig.ports;
+
+    const defaultPath = path.join(getConfig(remote.app).storage.dataDir);
+    const [folderPath, setPath] = useState(defaultPath);
+
+    const defaultEth1URL = "http://127.0.0.1:8545";
+    const [eth1Url, setEth1URL] = useState(defaultEth1URL);
+
+    const defaultRpcPort = "5052";
     const [rpcPort, setRpcPort] = useState(defaultRpcPort);
-    const defaultLibp2pPort = ports[0].local;
+
+    const defaultLibp2pPort = "9000";
     const [libp2pPort, setLibp2pPort] = useState(defaultLibp2pPort);
 
+    const defaultDiscoveryPort = "9000";
+    const [discoveryPort, setDiscoveryPort] = useState(defaultLibp2pPort);
```

Comment:
I think we should have default value extracted in some config related to lighthouse

Reply:
another option is to have its own config for all validators and if the user gonna request defaults by validator then we can implement

Reply:
oh but default values are related to beacon node (not validator) like lighthouse, prysm, nimbus, teku etc

Reply:
oh, I mean for beacon nodes :see_no_evil:

Reply:
I still don't get what you meant :rofl: